plugin_tests/annotations_test.py

```diff
@@ -439,6 +442,49 @@ def testGetAnnotation(self):
         self.assertLess(elements[0]['width'] * elements[0]['height'],
                         elements[-1]['width'] * elements[-1]['height'])
 
+    def testAnnotationCopy(self):
+        # create annotation on an item
+        annotModel = self.model('annotation', 'large_image')
+        file = self._uploadFile(os.path.join(
+            os.environ['LARGE_IMAGE_DATA'], 'sample_image.ptif'))
+        itemSrc = self.model('item').load(
```

Comment:
This can be changed to `itemSrc = Item().load(`.  Since you are using the new model semantics, let's use them in all new and changed code.

Similarly, we can import the annotation model via `from girder.plugins.large_image.models.annotation import Annotation`, and then use `Annotation()` instead of `annotModel`.